app/shared/ui-wizard/src/commonMain/kotlin/ui/wizard/navigation/WizardController.kt

```diff
@@ -75,6 +76,39 @@ class WizardController() {
         }
     }
 
+    internal suspend fun subscribeNavDestChanges(
```

Comment:
```suggestion
    internal suspend fun collectNavDestChanges(
```

subscribe 听起来像是注册回调, 但实际行为是 flow collect

Reply:
先看 https://github.com/open-ani/animeko/pull/1572#discussion_r1950776581